frontend/public/components/pod.tsx

```diff
@@ -551,7 +552,22 @@ const PodGraphs = requirePrometheus(({ pod }) => {
   );
 });
 
-export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => <Status status={podPhase(pod)} />;
+export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => {
+  const status = podPhase(pod);
+  const unschedulableCondition = pod.status?.conditions?.find(
+    (condition) => condition.reason === 'Unschedulable',
```

Comment:
```suggestion
    (condition) => condition.reason === 'Unschedulable' && condition.status === 'True',
```

Reply:
This actually breaks the check as `condition.type === PodScheduled`.  Think we want `(condition) => condition.reason === 'Unschedulable' && condition.status === 'False',`
<img width="1338" alt="Screen Shot 2020-11-23 at 3 25 25 PM" src="https://user-images.githubusercontent.com/895728/100011712-2133eb00-2da0-11eb-90cb-e5fb0bfc7c54.png">